src/elision.rs

```diff
@@ -5,6 +5,8 @@
 // http://opensource.org/licenses/MIT>, at your option. This file may not be
 // copied, modified, or distributed except according to those terms.
 
+#[cfg(all(feature = "nightly", any(target_arch = "x86", target_arch = "x86_64")))]
```

Comment:
Maybe rename the feature to "hardware-lock-elision" now or in a future release?

Reply:
In a future version I will just have this always enabled by default.

Reply:
Don't some processors have a bug causing "unpredictable system behavior" (aka crashes or worse) when using hardware lock elision?

Reply:
That's actually a good point, I wasn't aware that HLE was broken on all currently available CPUs.